src/app/views/search-route/search-route.component.ts

```diff
@@ -78,7 +78,7 @@ export class SearchRouteComponent implements OnInit, AfterViewInit {
     // https://developers.google.com/maps/documentation/javascript/place-data-fields?hl=en_US
     autocompleteOrigin.setFields(['place_id', 'name']);
     // When the user selects an address from the drop-down
-    google.maps.event.addListenerOnce(autocompleteOrigin, 'place_changed', () => {
+    google.maps.event.addListener(autocompleteOrigin, 'place_changed', () => {
```

Comment:
Would love to hear a "5 min explanation" of addListenerOnce vs addListener in our next meeting, if you don't mind. :)